<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VMHAEditPage</class>
 <widget class="QWidget" name="VMHAEditPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelInfo">
     <property name="text">
      <string>Configure high availability and startup settings for this virtual machine.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxHA">
     <property name="title">
      <string>High Availability</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelRestartPriority">
        <property name="text">
         <string>Restart priority:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBoxRestartPriority">
        <item>
         <property name="text">
          <string>Do not restart</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Restart</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Best-effort</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxStartupOptions">
     <property name="title">
      <string>Startup Options</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="labelStartOrder">
        <property name="text">
         <string>Start order:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spinBoxStartOrder">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>99999</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelStartDelay">
        <property name="text">
         <string>Start delay (seconds):</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spinBoxStartDelay">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>99999</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
